charts/tfy-karpenter-config/templates/controlplane/karpenter-control-plane-ec2nodeclass.yaml

```diff
@@ -40,7 +40,7 @@ spec:
   amiSelectorTerms:
   {{- .Values.karpenter.controlPlaneNodeTemplate.amiSelectorTerms | toYaml | nindent 2 }}
   {{- end }}
-  {{- if eq .Values.karpenter.controlPlaneNodeTemplate.amiFamily "Bottlerocket" }}
+  {{- if eq (include "criticalNodeClass.inferredAmiFamily" .) "bottlerocket" }}
```

Comment:
Looks incorrect - should be controlPlaneNodeTemplate

Reply:
I must swapped these by mistake, fixing